apps/docs/package.json

```diff
@@ -45,5 +45,8 @@
     "webpack-cli": "^3.3.11",
     "webpack-dev-middleware": "^3.7.2",
     "webpack-hot-middleware": "^2.25.0"
+  },
+  "dependencies": {
+    "@hods/pagination": "workspace:^0.1.8"
```

Comment:
You shouldn't need this.

Reply:
Again, not sure why this was added. Have removed it now